esbuilder/lit-connect-modal/src/modal.js

```diff
@@ -0,0 +1,163 @@
+import MicroModal from 'micromodal';
+import css from './modal.css';
+import {
+  rawListOfWalletsArray,
+  metaMaskSingle,
+  coinbaseSingle,
+} from './helpers/walletList.js';
+import providerMethods from './helpers/providerMethods.js';
+
+export default class LitConnectModal {
+  constructor({ providerOptions }) {
+    this.dialog = MicroModal;
+    this.closeAction = undefined;
```

Comment:
can this be passed as an option?

Reply:
yes we can, we can only a new pr for this. this pr is solely to fix the wc issue with the existing code